boards/xtensa/esp32/doc/esp32.rst

```diff
@@ -121,31 +121,29 @@ the tool.
 Using JTAG
 ==========
 
-In addition to the SoC vendor and SDK, this board also requires a build of
-OpenOCD with patches supporting the SoC.  The source code and build
-instructions can be obtained from GitHub:
+As with much custom hardware, the ESP-32 moudules require patches to
+OpenOCD that are not upstream.  Espressif maintains their own fork of
+the project here.  By convention they put it in the ~/esp next to the
```

Comment:
Please provide a link to the repository here. Might be a good idea to mark `~/esp` with backticks so it's typed with fixed width font.

Reply:
It's there in the clone command.  Seems needless to repeat it in text.  I'll admit RST is not my thing, will add the markup.

Reply:
RST will be rendered as HTML, so this will become a link.